robottelo/hosts.py

```diff
@@ -872,6 +872,18 @@ def enable_dnf_proxy(self, hostname, scheme=None, port=None):
         logger.info(f'Configuring {hostname} HTTP proxy for dnf.')
         self.execute(cmd)
 
+    def enable_ipv6_rhsm_proxy(self):
+        """Execute procedures for enabling rhsm and dnf IPv6 HTTP Proxy"""
+        if self.ipv6:
+            url = urlparse(settings.http_proxy.http_proxy_ipv6_url)
+            self.enable_rhsm_proxy(url.hostname, url.port)
+
+    def enable_ipv6_dnf_proxy(self):
+        """Execute procedures for enabling rhsm and dnf IPv6 HTTP Proxy"""
+        if self.ipv6:
+            url = urlparse(settings.http_proxy.http_proxy_ipv6_url)
+            self.enable_dnf_proxy(url.hostname, url.scheme, url.port)
+
```

Comment:
Why did you introduce these new functions? They seem not used anywhere but in the old `enable_ipv6_dnf_and_rhsm_proxy`.

(also, the doc strings are wrong, as they now do only one, RHSM *or* DNF)

Reply:
I wanted to separate them so that users who only want to enable rhsm or dnf can do so without enabling the other. To make code more simpler. It's not used at other places now but it most probably will be going forward and I think it's better to have them separate.